polar-core/tests/integration_tests.rs

```diff
@@ -1560,20 +1560,22 @@ fn test_and_or_warning() -> TestResult {
     p.load_str("f(x) if (\n\t    x = 1 and  x > 1) or x < 3;")?;
 
     p.clear_rules();
+    p.load_str("f(x) if x = 1 and x > 1 or x < 3;")?;
+    let msg = p.next_message().unwrap();
+    assert!(matches!(&msg.kind, MessageKind::Warning));
+    assert_eq!(
+        &msg.msg,
+        "Expression without parentheses could be ambiguous. \nPrior to 0.20, `x and y or z` would parse as `x and (y or z)`. \nThis was changed in 0.20 to match other languages. \n\n\n001: f(x) if x = 1 and x > 1 or x < 3;\n             ^"
```

Comment:
nit on the warning message
```suggestion
        "Expression without parentheses could be ambiguous. \nPrior to 0.20, `x and y or z` would parse as `x and (y or z)`. \nAs of 0.20, it parses as `(x and y) or z`, matching other languages. \n\n\n001: f(x) if x = 1 and x > 1 or x < 3;\n             ^"
```

Reply:
👍